src/database/models/categories.rs

```diff
@@ -29,10 +29,10 @@ pub struct ReportType {
 }
 
 #[derive(Serialize, Deserialize)]
-pub struct DonationPlatform {
-    pub id: DonationPlatformId,
-    pub short: String,
+pub struct LinkPlatform {
+    pub id: LinkPlatformId,
     pub name: String,
+    pub donation: bool,
```

Comment:
is this bool needed? I assume it's for legacy compat but I feel like there's a better way to do this

Reply:
Like maybe we could hardcode? or do you think that's worse

Reply:
I kinda feel like this is the structurally to hardcoding but without having to actually hardcode it
It might also be useful for the frontend at some point to know which links are donation links without hardcoding (ie: 'see donation links' or w/e depending on how it 
I'm for it being this way, but dont feel that strongly about it so can change it if you prefer